rebar.config

```
@@ -25,6 +25,13 @@
     warn_missing_spec_all
 ]}.
 
+{deps, [
+    {damsel,
+        {git, "https://github.com/rbkmoney/damsel.git",
+        {branch, "epic/test_trying_to_generate_as_external_dep"}}
```

Я чёт только сегодня осознал, что ты можешь здесь просто сослаться на
```suggestion
        {branch, "release/erlang/master"}}
```

---

...И сделать `rebar3 upgrade` в контейнере после этого.

---

> `rebar3 upgrade` в контейнере

можешь пояснить , что за контейнер?